modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -58,8 +58,30 @@ class FeatureFloodCount :
   virtual void finalize() override;
   virtual Real getValue() override;
 
+  /// Returns the total feature count (active and inactive ids, useful for sizing vectors)
+  virtual std::size_t getTotalFeatureCount() const;
```

Comment:
Yeah - I'm moving to being more type correct. "sizes", "counts" and "indices" that interact with vectors are std::size_t. I'm trying to use `std::size_t` for those things but I'm going to continue using `unsigned int` for the actual feature_id for the time being. That's the difference throughout.